app/components/autocomplete/autocomplete.js

```diff
@@ -55,6 +62,33 @@ export default class Autocomplete extends PureComponent {
         this.setState({commandCount});
     };
 
+    componentWillMount() {
+        this.keyboardDidShowListener = Keyboard.addListener('keyboardDidShow', (e) => this.keyboardDidShow(e));
```

Comment:
If you define the methods below as arrow functions, you can save having to do that on this line like
```
this.keyboardDidShowListener = Keyboard.addListener('keyboardDidShow', this.keyboardDidShow);
...
keyboardDidShow = (e) => {
    ...
};
```